core-services/pdc/src/main/java/org/silverpeas/core/pdc/pdc/service/DefaultPdcClassifyManager.java

```diff
@@ -200,14 +153,9 @@ public void registerAxis(Connection con, int axisId) throws PdcException {
   public void unregisterAxis(Connection con, int axisId) throws PdcException {
     try {
       List<Integer> alDeletedPositionIds = classifyEngine.unregisterAxis(con, axisId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "alDeletedPositionIds" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)